src/components/manage/Contents/ContentsUploadModal.jsx

```diff
@@ -252,7 +262,12 @@ class ContentsUploadModal extends Component {
                       </Table.Cell>
                       <Table.Cell>
                         {file.type.split('/')[0] === 'image' && (
-                          <Image src={file.preview} height={60} />
+                          <Image
+                            src={`${flattenToAppURL(
```

Comment:
@tiberiuichim Latest react-dropzone package hasn't has `preview` property. How do we internally handle image previews when images are coming not from `@image` endpoints?

Reply:
https://github.com/plone/volto/blob/88b6ea08580436f7a3614f07cab52711238acd97/src/components/manage/Widgets/FileWidget.jsx#L66

Reply:
4bee4f2 fixes it.